uobtheatre/productions/models.py

```diff
@@ -166,10 +168,18 @@ def capacity_remaining(self, seat_group: SeatGroup = None):
             return self.total_capacity(seat_group=seat_group) - len(
                 self.tickets(seat_group=seat_group)
```

Comment:
Need to think about how we are dealing with draft bookings - This is probably gonna be another ticket

Reply:
Atm `booking.tickets()` is gonna give all the tickets but we will need to extend that to filter on status 
default: paid and some/all of the in-progress (depending on how we handle the logic). Im expecting we add an inactive status or something along these lines or look at the time it was last active?

Reply:
#47